core/src/test/java/org/arquillian/smart/testing/configuration/ConfigurationTest.java

```diff
@@ -40,6 +40,7 @@ public void should_load_configuration_with_default_values_if_property_is_not_spe
         expectedConfiguration.setScm(scm);
         expectedConfiguration.setReport(report);
         expectedConfiguration.setAutocorrect(true);
+        expectedConfiguration.setCustomStrategies(new String[]{"smart.testing.strategy.cool=org.arquillian.smart.testing:strategy-cool:1.0.0"});
```

Comment:
I understand your point, for now and to not make this PR longer, I'd advocate for leaving this file for internal usage.

Reply:
I am not sure if "not keeping PR longer" is the right success criteria - if this change does not mean a lot of work shipping better user experience is more important from my point of view than something which might be quite confusing.

Reply:
I think we tend to add more features in PRs that the ones it describes originally. In my opinion a PR should implement what it mentions, if during the process we see that there is an internal design problem then I'd open a new PR. But it is my opinion.

Reply:
I don't think it's adding more features, I think it's striving to have a good feature in the first place.

Reply:
well we can drop this file and add as default initial values to strategies the content of this file. Personally since this file is just used internally I'd leave as is, and for now `strategies.properties` is used for internal/default strategies.